### PR TITLE
Nerfing Unarmed Weapons/Perks and Rubber Bullets

### PR DESCRIPTION
**Pull request by AsterixCodix**
## About The Pull Request

Traits adjusted: 
- Straight Shooter, from 50% down to 15%, but cost reduced to 3
- Iron Fist, from 12 upper and 2 lower, to 10 upper and 4 lower
- Steel Fist, from 14 upper and 4 lower, to 12 upper and 6 lower

**Rubber** ammo nerfed:
- .22
- .45ACP
- 10mm
- .308/7.62
- Rubber Shotgun Pellets
- Rubber Shotgun Slugs

Nerfs to Unarmed
- Sappers nerfed from 20 additional Stamina Damage to 10
- Spiked Knuckles lost their additional AP.
- Reduces base stamina damage for punching something by 25%

## Pre-Merge Checklist
- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

## Changelog
:cl:
balance: nerf to rubber .22
balance: nerf to rubber .45ACP
balance: nerf to rubber 10mm
balance: nerf to rubber .308/7.62
balance: nerf to rubber shotgun pellets
balance: nerf to rubber shotgun slugs
balance: nerf to Sappers
balance: nerf to Spiked Knuckles
balance: nerf to base punching stamina damage
balance: nerf to Straight Shooter
balance: halved Straight Shooter cost
balance: nerf to Iron Fist upper damage
balance: buff to Iron Fist lower damage
balance: nerf to Steel Fist upper damage
balance: buff to Steel Fist lower damage
/:cl:

### CODE REVIEW

**Comment by X404ErrorX** [comment]
The melee combat is already bad in itself and this pr is going to finish it off.

**Comment by Teragenic** [comment]
This doesn't hurt? Melee combat all that much, it NERFS unarmed damage which nerfs stam combat. Much needed stam combat nerf. 

Now I just hope they are still VIABLE for actual MP/Slave keeping use.

**Comment by SirSwagmeyer** [comment]
Straight Shooter should cost 2 instead of 3 if you're going to nerf it that hard. 15% is barely worth the investment for a whopping three points, and it's likely going to kill the trait entirely. I feel that 25% with a cost of 3 is balanced enough. Hell, I don't think the change should happen at all, but that's just me. Six entire trait points for the buff seemed reasonable. 

I also disagree with the AP removal of the Spiked Knuckles. The Spiked Knuckles do absolutely nothing aside from deal 2 more damage compared to the brass knuckles. The armor-piercing was the entire point of Spiked Knuckles existing, and I don't think it's a fair PR to remove that completely when this PR's already gutted unarmed on a widespread level.

File: code/game/objects/items/melee/f13onehanded.dm [hunk 1]
```diff
@@ -704,11 +704,11 @@ obj/item/melee/onehanded/knife/switchblade
...
 			H.dna.species.punchdamagelow = 1
 		if(HAS_TRAIT(user, TRAIT_IRONFIST))
-			H.dna.species.punchdamagehigh = 12
-			H.dna.species.punchdamagelow = 2
-		if(HAS_TRAIT(user, TRAIT_STEELFIST))
-			H.dna.species.punchdamagehigh = 14
+			H.dna.species.punchdamagehigh = 10
 			H.dna.species.punchdamagelow = 4
+		if(HAS_TRAIT(user, TRAIT_STEELFIST))
+			H.dna.species.punchdamagehigh = 12
```
Comment (Zero-Percent-Angel) [inline code/game/objects/items/melee/f13onehanded.dm:710]: If you're going to change these, there is somewhere else you have to change them too. Possibly on the traits where they're granted

**Pull request merged**